gtfs_realtime_translators/translators/wcdot_bus.py

```diff
@@ -0,0 +1,43 @@
+import json
+from gtfs_realtime_translators.factories import TripUpdate, FeedMessage
+
+
+class WcdotGtfsRealTimeTranslator:
+    def __init__(self, stop_id=None):
+        if stop_id is None:
+            raise ValueError('stop_id is required.')
+        self.stop_id = stop_id
+        self.filtered_stops = None
+
+    def __call__(self,data):
+        json_data = json.loads(data)
+        entities = json_data["entity"]
```

Comment:
Not sure of the data you get as input but don't you need any error handling to handle incorrect dataor if "entity" is not found in json_data.

Reply:
I agree. The reason for a lack of error handling is because the class calling this translator is handling errors/exceptions, so I figured it was ok to keep this as is. Curious to hear your thoughts on this

Reply:
So far as the calling class is handling the errors/exceptions or passing only validated data as input, it's fine.
If the calling class is not handling it, you might want to log and continue Or throw exception and stop depending on the nature of the script/data you are processing.

Reply:
We should get a FailedProcessingException in Inbound with a stack trace logged to DD, so I think we should be fine as is

Reply:
One way to break this problem down a little further @JorgeBillini is to focus on the responsibilities and see if error handling here/there makes sense to the domain/responsibility of the given component. 

I think there are two types of errors / validations that need to occur and perhaps one or both is already implemented. Data/Type-specific validations (is the JSON valid), and domain-specific validations (is this entity in the data supporting how we want to use it). The latter would need to belong in domain-specific objects (preferably models) and it sounds from this discussion that is what we're doing here. 

If one of those validations is missing we should look into adding it. Example: if we're not asserting the JSON object is semantic, throw an error.